src/Aspire.Cli/Program.cs

```diff
@@ -198,6 +198,11 @@ private static void AddInteractionServices(HostApplicationBuilder builder)
                     provider.GetRequiredService<IExtensionBackchannel>(),
                     extensionPromptEnabled);
             });
+
+            // If the CLI is being launched from the aspire extension, we don't want to log to the console.
```

Comment:
> we don't want to log to the console

What makes it so this doesn't go to the console?

Reply:
I clarified the comment @eerhardt